Dockerfile

```
@@ -8,19 +8,24 @@ RUN chmod +x gradlew
+
 COPY --from=build /app/build/libs/*.jar app.jar
+COPY --from=build /app/config/application-prod.properties /home/app/config/
 
 RUN chown -R 1000:1000 /home/app
 
 ENV HOME=/home/app
 
 USER 1000
-ENTRYPOINT ["java", "-Dspring.profiles.active=prod", "-jar", "app.jar"]
+ENTRYPOINT ["java",
+  "-Dspring.profiles.active=prod",
```

Hadolint (2.14.0)

[error] unexpected '"'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)